i2c/bmp280_i2c/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_executable(bmp280_i2c
+        bmp280_i2c.c
+        )
+
+# Pull in our get you started dependencies
```

Comment:
Needs tweaking :wink:

Reply:
lol...

Reply:
It's like a virus that keeps popping back up every time you think you've killed it off :rofl: